src/main/scala/sbtecr/AwsEcr.scala

```diff
@@ -13,19 +13,29 @@ private[sbtecr] object AwsEcr extends Aws {
 
   def domain(region: Region, accountId: String) = s"${accountId}.dkr.ecr.${region}.${region.getDomain}"
 
-  def createRepository(region: Region, repositoryName: String)(implicit logger: Logger): Unit = {
+  def createRepository(region: Region, repositoryName: String, repositoryPolicyTextOption : Option[String])(implicit logger: Logger): Unit = {
+    val client = ecr(region)
     val request = new CreateRepositoryRequest()
     request.setRepositoryName(repositoryName)
 
     try {
-      val result = ecr(region).createRepository(request)
+      val result = client.createRepository(request)
       logger.info(s"Repository created in ${region}: arn=${result.getRepository.getRepositoryArn}")
+      repositoryPolicyTextOption.foreach(setPolicy(client, repositoryName, _))
     } catch {
       case e: RepositoryAlreadyExistsException =>
         logger.info(s"Repository exists: ${region}/${repositoryName}")
     }
   }
 
+  def setPolicy(ecr : AmazonECR, repositoryName : String, repositoryPolicyText : String)(implicit logger: Logger) : Unit = {
```

Comment:
1. Since the method takes `ecr: AmazonECR` as an argument it should be kept `private`. 
2. Please remove the extra space before `:` to keep the style consistent.